src/test/java/com/spotify/docker/client/DefaultDockerClientTest.java

```diff
@@ -455,7 +455,8 @@ public void testPullPrivateRepoWithoutAuth() throws Exception {
   public void testBuildImageIdWithBuildargs() throws Exception {
     requireDockerApiVersionAtLeast("1.21", "build args");
 
-    final String dockerDirectory = Resources.getResource("dockerDirectoryWithBuildargs").getPath();
```

Comment:
See reply in [comment below](https://github.com/spotify/docker-client/pull/780#issuecomment-306270015) (i didn't notice this reply box :) )